readme.md

```diff
@@ -1 +1,105 @@
+# remark-language-server
 
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+A language server to lint and format markdown files with **[remark][]**.
+
+## Contents
+
+- [What is this?](#what-is-this)
+- [When should I use this?](#when-should-i-use-this)
+- [Install](#install)
+- [Use](#use)
+- [Compatibility](#compatibility)
+- [Contribute](#contribute)
+- [License](#license)
+
+## What is this?
+
+This package is a [language server][] which can lint and format markdown files
+using remark.
+
+You can choose from the 150+ plugins that already exist or make your own.
+
+See [the remark monorepo readme][remark] for info on what the remark ecosystem
+is.
+
+## When should I use this?
+
+You can use this package when you want to enhance your editor with linting and
+formatting of markdown files.
+Some editors can consume this package directly, others need a plugin in order to
+consume this package.
+
+## Install
+
+This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
+In Node.js (version 12.20+, 14.14+, or 16.0+), install with [npm][]:
```

Comment:
🤔 would it make sense to specify which LSP version this uses? (I'm guessing 3.16? https://microsoft.github.io/language-server-protocol/specifications/specification-current/)

Reply:
Yes definitely! (I believe 3.16 is correct) Although maybe it would be better to document this at `unified-language-server` instead?

Perhaps it also makes sense to document which commands have been implemented.

Reply:
See https://github.com/unifiedjs/unified-language-server/pull/16